src/coreclr/jit/lclvars.cpp

```diff
@@ -564,8 +564,7 @@ void Compiler::lvaInitRetBuffArg(InitVarDscInfo* varDscInfo, bool useFixedRetBuf
             info.compCompHnd->getMethodSig(info.compMethodHnd, &sigInfo);
             assert(JITtype2varType(sigInfo.retType) == info.compRetType); // Else shouldn't have a ret buff.
 
-            info.compRetBuffDefStack =
-                (info.compCompHnd->isStructRequiringStackAllocRetBuf(sigInfo.retTypeClass));
+            info.compRetBuffDefStack = (info.compCompHnd->isStructRequiringStackAllocRetBuf(sigInfo.retTypeClass));
             if (info.compRetBuffDefStack)
```

Comment:
Wait, my changes didn't affect this one (it was the suggested patch). CI didn't pass in the main branch, did it? 🦆

Reply:
what do you mean? I do not think that the jit-format job deleted `== TRUE);` here.

Reply:
Ohhh, right, sorry, I'm blind XD. Yes, it's completely alright.